app/src/main/java/com/sapuseven/untis/ui/activities/InfoCenter.kt

```diff
@@ -438,3 +556,196 @@ private fun formatOfficeHourTime(
 		endDateTime.toString(DateTimeFormat.shortTime())
 	)
 }
+
+
+@OptIn(ExperimentalMaterial3Api::class)
+@Composable
+private fun AbsenceFilterDialog(
+	initialOrder: AbsenceOrder,
+	onDismiss: (requestedAbsenceOrder: AbsenceOrder) -> Unit,
+) {
+	var absenceOrder by rememberSaveable(
+		saver = Saver(
+			save = { Json.encodeToString(it.value) },
+			restore = { mutableStateOf(Json.decodeFromString(it)) }
+		)
+	) { mutableStateOf(initialOrder) }
+	var dismissed by rememberSaveable { mutableStateOf(false) }
+
+
+	fun dismiss(absenceOrder: AbsenceOrder) {
+		onDismiss(absenceOrder)
+		dismissed = true
+	}
+
+	BackHandler(
+		enabled = !dismissed,
+	) {
+		dismiss(absenceOrder)
+	}
+
+	Scaffold(
+		topBar = {
+			CenterAlignedTopAppBar(
+				title = { Text(text = "Filter Absencses") },
+				navigationIcon = {
+					IconButton(onClick = {
+						dismiss(absenceOrder)
+					}) {
+						Icon(
+							imageVector = Icons.Outlined.Close,
+							contentDescription = stringResource(id = R.string.all_close)
+						)
+					}
+				}
+			)
+		}
+	) { innerPadding ->
+		VerticalScrollColumn(paddingValues = innerPadding) {
+			PreferenceCategory(title = "") {
```

Comment:
Omit `PreferenceCategory` for "root preferences". All it does is add the title as a heading, so here it's not needed.